### PR TITLE
Add "Simply Logical" to the gallery

### PR DESCRIPTION
**Pull request by So-Cool**
Add [Simply Logical](https://book.simply-logical.space/) to the [Jupyter Book gallery](https://executablebooks.org/en/latest/gallery.html).

### CODE REVIEW

**Review by choldgraf — APPROVED** [review]
Many thanks for the addition!!

File: docs/gallery.yml [hunk 1]
```diff
@@ -322,3 +322,7 @@
   website: https://pyhf.github.io/pyhf-tutorial/
   repository: https://github.com/pyhf/pyhf-tutorial
   image: https://github.com/scikit-hep/pyhf/raw/master/docs/_static/img/pyhf-logo.png
+- name: "Simply Logical – Intelligent Reasoning by Example"
+  website: https://book.simply-logical.space/
+  repository: https://github.com/simply-logical/simply-logical
+  image: https://raw.githubusercontent.com/simply-logical/simply-logical/master/src/img/SL.svg
```
Comment (chrisjsewell) [inline docs/gallery.yml:328]:
```suggestion
```

**Comment by chrisjsewell** [comment]
Thanks @So-Cool however, would you mind fixing the merge conflict (or you/I can just open a new PR)

**Comment by So-Cool** [comment]
All done, @chrisjsewell.

**Review by chrisjsewell — APPROVED** [review]
Cheers!

**Pull request merged**